index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<!--Head section of HTML-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title> Horiseon </title>
+</head>
+
+<!--Body section of HTML-->
+
+<body>
+    <!--Header section-->
+    <header class="header">
+        <h1>Hori<span class="seo">seo</span>n</h1>
+        <!--Listless navigation bar-->
+        <nav>
+            <ul>
+                <a class="nav-btn" href="#search-engine-optimization">Search Engine Optimization</a>
+                <a class="nav-btn" href="#online-reputation-management">Online Reputation Management</a>
+                <a class="nav-btn" href="#social-media-marketing">Social Media Marketing</a>
+            </ul>
+        </nav>
+    </header>
+
+    <!--Hero section, to declare an oversized banner at the top of the web page-->
+    <section class="hero"></section>
+
```

Comment:
I would of left the image here, rather than moving it to the CSS. Whilst it does work, it doesn't allow you to add the necessary `alt` tag.